src/journal/views.py

```diff
@@ -137,18 +137,17 @@ def funder_articles(request, funder_id):
     if redirect:
         return redirect
 
-    pinned_articles = [pin.article for pin in models.PinnedArticle.objects.filter(
-        journal=request.journal)]
-    pinned_article_pks = [article.pk for article in pinned_articles]
+    artile_funding_objects = submission_models.ArticleFunding.objects.filter(
+        fundref_id=funder_id,
+        article__journal=request.journal,
+        article__date_published__lte=timezone.now(),
+        article__section__pk__in=filters,
```

Comment:
do we need to filter by `article__stage=sm_models.STAGE_PUBLISHED` as well? we always do it in tandem with `article__date_published`